state/data/actions.js

```diff
@@ -9,3 +10,14 @@ export const mergeListsList = (lists) => ({
   type: types.MERGE_LISTS_LIST,
   lists
 });
+
+export const mergeMovie = (movie) => ({
```

Comment:
If I remember you already removed this function

Reply:
My fault. Fixed